template.distillery/README.md

```diff
@@ -222,3 +228,24 @@ you update the server:
 ```
 make APP_SERVER=http://${YOUR_SERVER} APP_REMOTE={yes|no} chcp
 ```
+
+## Use Makefile.local file.
+
+You need to define `APP_REMOTE` and `APP_SERVER` each time you want to build
+mobile applications or to update it.
+
+If you don't want to pass every time the variables `APP_SERVER` and
+`APP_REMOTE`, you can change the values of these variables in
+`Makefile.local.example` and rename this file in `Makefile.local`. With this
+method, `APP_REMOTE` and `APP_SERVER` variables are not mandatory when you build
```

Comment:
* The variables `APP_REMOTE` and `APP_SERVER` are not mandatory
* _With this method, ..._ -> _This way, ..._ (simpler)